jupyter/Scale Output Example.ipynb

```diff
@@ -0,0 +1,182 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Simple Scalar Output Example\n",
+    "\n",
+    "In this example, we will define an integer parameter called `Integer input` and will output a floating point called `My scalar output` that can be bound to a numeric dashboard tile."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import scrapbook as sb \n",
+    "from random import *"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Parameters\n",
+    "\n",
+    "The parameters block is where you define all of the inputs and outputs to the notebook.  If you look at __Property Inspector >> Advanced Tools >> Cell Metadata__, while the `# Parameters` code cell block is selected, you will see the json that describes the inputs and outputs. Note that when you edit that json, you must click the small check mark under \"Cell Metadata\" to save the changes. In this example you can see we have configured a numeric input and a numeric output."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {
+    "papermill": {
+     "parameters": {
+      "numeric_param": 1
+     }
+    },
+    "systemlink": {
+     "namespaces": [
+      "ni-testmanagement"
+     ],
+     "outputs": [
+      {
+       "display_name": "Input plus fraction",
+       "id": "scalar_output",
+       "type": "scalar"
+      }
+     ],
+     "parameters": [
+      {
+       "display_name": "Number",
+       "id": "numeric_param",
+       "type": "number"
+      }
+     ],
+     "version": 2
+    },
+    "tags": [
+     "parameters"
+    ]
+   },
+   "outputs": [],
+   "source": [
+    "# Parameters\n",
+    "numeric_param = 1 # default value used when input is not provided by Dashboard\n",
+    "\n",
+    "result = [] # container for all of the notebooks results"
```

Comment:
Papermill works by injecting a new cell containing parameter value overrides directly below the cell marked `parameters`. Therefore, it's a good practice to keep any code that's not defining a default value for a parameter outside of the parameters cell. You're technically safe in this particular instance, but if this line depended on the parameter above, you can end up with some really weird (and hard to debug) behavior since that code would be referencing the default value, not the passed-in value.

Reply:
Also this line feels like it belongs in the "Results" cell a few cells down.